ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/daemon_terminattr.schema.yml

```
@@ -89,11 +89,11 @@ keys:
             cvsourceip:
               type: str
               description: |
-                Set source IP address in case of in-band managament
+                Set source IP address in case of in-band management
```

I know this is not related to this PR but lets take the opportunity to fix it
```suggestion
                Set source IP address in case of in-band management.
```